.github/workflows/examples-java-push.yml

```
@@ -55,3 +55,20 @@ jobs:
           name: examples-java-reports
           path: '**/build/reports/*'
           retention-days: 5
+  deploy:
+    runs-on: ubuntu-18.04
```

This is needed to pick up the environment variables for the Environment we're executing against ([docs](https://docs.github.com/en/actions/deployment/targeting-different-environments/using-environments-for-deployment#using-an-environment))
```suggestion
  deploy:
    environment: sandbox
    runs-on: ubuntu-18.04
```